### PR TITLE
Jv shop tomcat

### PR DESCRIPTION
**Pull request by vladkuzya**

### CODE REVIEW

File: src/main/java/mate/academy/dao/UserDao.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package mate.academy.dao;
+
+import mate.academy.model.User;
+
+public interface UserDao extends GenericDao<User, Long> {
+
```
Comment (Borisko130) [inline src/main/java/mate/academy/dao/UserDao.java]: You can remove empty line here

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Влад
+  Date: 08.09.2020
+  Time: 12:53
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (Borisko130) [inline src/main/webapp/WEB-INF/views/index.jsp]: Remove autogenerated comments from index.jsp.

File: src/main/java/mate/academy/dao/UserDao.java (hunk 1 above)
Comment (vladkuzya) [reply src/main/java/mate/academy/dao/UserDao.java]: done)

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 2 above)
Comment (vladkuzya) [reply src/main/webapp/WEB-INF/views/index.jsp]: done)

**Review by Borisko130 — APPROVED** [review]

**Pull request closed without merging**